src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -479,7 +479,7 @@ object TezosDatabaseOperations extends LazyLogging {
     * @return a database action that stores both arguments and return a tuple of the row counts inserted
     */
   def writeAccountsAndCheckpointDelegates(
-      accounts: List[(BlockTagged[Map[AccountId, Account]], List[AccountsHistoryRow])],
+      accounts: List[(BlockTagged[Map[AccountId, Account]], List[Tables.AccountsRow])],
```

Comment:
We might want to only pass in the block-tagged map as a method parameter here, if the existing rows are not needed.
In fact, it's not clear at all, if we look at the method description, what should we pass in for this list of accounts rows... 😉

Reply:
updated docs for that